lib/kube/proxy/cluster_details.go

```diff
@@ -147,6 +158,7 @@ func newClusterDetails(ctx context.Context, cfg clusterDetailsConfig) (_ *kubeDe
 				k.rwMu.Lock()
 				k.kubeCodecs = codecFactory
 				k.rbacSupportedTypes = rbacSupportedTypes
+				k.isClusterOffline = false
```

Comment:
Why not have the same `var isClusterOffline bool` logic here as above?

Reply:
We don't need it. If we build the logic once we can re-use.
Here we are trying to include new CRDs that were installed in the meanwhile but it's not something that prevents users to access the cluster.